packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -633,6 +633,10 @@ class _NestedScrollCoordinator implements ScrollActivityDelegate, ScrollHoldCont
   late _NestedScrollController _outerController;
   late _NestedScrollController _innerController;
 
+  bool get outOfRange {
+    return (_outerPosition?.outOfRange ?? false) || _innerPositions.any((_NestedScrollPosition position) => position.outOfRange);
+  }
```

Comment:
Not a big deal whatsoever, but it would be nice to stay within the [80 characters style guideline](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#prefer-a-maximum-line-length-of-80-characters):

```dart
bool get outOfRange {
  if (_outerPosition?.outOfRange ?? false) {
    return true;
  }
  return _innerPositions.any((_NestedScrollPosition position) => position.outOfRange);
}
```

Reply:
Honestly, this is very mild, compared to [other things I've seen](https://github.com/flutter/flutter/pull/143812#discussion_r1496810160). I'll go ahead and throw on `autosubmit` :)